tests/unit/apps/test_subordinate.py

```diff
@@ -60,29 +60,40 @@ def test_generate_upgrade_plan(status):
 
 
 @pytest.mark.parametrize(
-    "channel, throws_exception",
+    "channel",
     [
-        ("ussuri/stable", False),
-        ("victoria/stable", False),
-        ("xena/stable", False),
-        ("yoga/stable", False),
-        ("wallaby/stable", False),
-        ("wallaby/edge", False),
-        ("focal/edge", True),
-        ("latest/edge", True),
-        ("latest/stable", True),
-        ("something/stable", True),
+        ("ussuri/stable"),
```

Comment:
Parameterize on a single variable doesn't need the parenthesis. Could you please remove them? (and also for the `invalid` test function)
```
@pytest.mark.parametrize( "channel", ["ussuri/stable", "victoria/stable", ...])

Reply:
done